test/spawn-options.spec.js

```diff
@@ -137,11 +137,16 @@ describe('Spawn options', () => {
           }
 
           f.spawn(options, (err, _ipfsd) => {
+            ipfsd = _ipfsd
             expect(err).to.not.exist()
             expect(_ipfsd).to.exist()
             expect(_ipfsd.api).to.not.exist()
 
-            ipfsd = _ipfsd
+            // proc nodes don't reuse initialized repos
+            if (fOpts.type !== 'proc') {
```

Comment:
We should add this functionality to `proc` nodes - there is no reason they can't do that. But I'm fine if its in another PR. Could you file an issue to add this please?

Reply:
Ahh cool. I really haven't even looked at in-proc nodes but this is an opportunity. I've filed an issue and am happy to work on it later. https://github.com/ipfs/js-ipfsd-ctl/issues/214